MyGet.ps1

```diff
@@ -0,0 +1,43 @@
+# Performs the build for MyGet and uploads the generated NuGet packages to
+# the "google-dotnet" feed.
+#
+# MyGet's build syncs the repo and subrepos, to build manually be sure to:
+# git submodule update --init ClientGenerator
+
+function PrintHeader($message) {
+    Write-Host "****************************************"
+    Write-Host $message
+    Write-Host "****************************************"
+}
+
+Write-Host ".NET Client Libraries, build shard $Env:BuildShard"
+
+# TODO(chrsmith): Document why we require VS v14 to build.
+[Environment]::SetEnvironmentVariable(
+    "VisualStudioVersion",
+    "14.0",
+    "User")
+
+$msbuild14 = "${Env:ProgramFiles(x86)}\MSBuild\14.0\Bin\MSBuild.exe"
```

Comment:
Yes, which is fine. `MyGet.ps1` is assumed to run on the MyGet build machine, which has its own unique configuration. We assume the configuration is in a certain state, e.g. it is a 64-bit OS, has Python 2.7 installed, etc.